consai_examples/consai_examples/field_observer.py

```diff
@@ -764,3 +764,56 @@ def _forward_robots_id(self, my_robot_id, my_robot_pos, robots_pos, robots_vel,
 
         return forward_robots_id
 
+    # シューターが相手ゴールに向かってシュートできるかを判定する関数
+    def get_shoot_point(self, my_robot_id):
+        # ゴールポストのx座標
+        goal_post_x = 6
+        # ゴールポストのy座標
+        goal_post_y = 0.9
+        # フィールド中央から見たゴールポストの座標
+        goal_post = [[goal_post_x, goal_post_y], [goal_post_x, - goal_post_y]]
+        # シュートを打つ目標点の座標
+        goal_center_target = [goal_post_x, 0]
+        # ロボットの実直径
+        robot_diameter = 0.2
```

Comment:
定数は大文字で表現してください。（90行目あたりの定数も大文字で表現すべきでした。。。）

```suggestion
        ROBOT_DIAMETER  = 0.2
```